test/statistics.jl

```diff
@@ -39,9 +39,14 @@
     # spatial variance
     σn = var(sdata[:Au])
     σs = var(sdata, :Au)
-    @test σn ≤ σs
+    @test 0 ≤ σs
     @test var(sdata)[:Au] ≈ σs
 
+    # mode heuristic
+    @test GeoStatsBase.mode_hsm([1]) == 1.0
+    @test GeoStatsBase.mode_hsm([1,2,2,3]) == 2.0
+    @test GeoStatsBase.mode_hsm(rand(1:6, 2111) + rand(1:6, 2111)) == 6.0
+
```

Comment:
Can we move these tests to the top of the test file as they are not of the same nature of the other tests? Or perhaps create a separate section in this file just for the heuristic functions?